### DISCUSSION TITLE
Does dynamic lifts include suction cup area?

### DISCUSSION THREAD

**Discussion by bergie5737** [sn4k3/UVtools#484] [Q&A] [answered]
Hi, Thanks for a great toolset! I am still new to it and the owner of a Nova Bene4. Sadly I see no way of changing acceleration on it, so I'm stuck with Dynamic lifts for now.

I have a model where due to a previous crack, I can only add a hole about 5mm up. The "flesh" area around the suction cup is much smaller than the suction cup, but coincidentally it is also the biggest surface on my model, so no big deal, but the next suction cup is a few layers with a large suction surface area, and little "flesh."

Anyway, so I am asking if the suction cup surface area is included in the model surface when calculating dynamic lifts or should I manually assign lift speeds to this 2nd large suction cup?

**Comment by sn4k3** [comment, marked as answer]
I'm not sure if i understood your question, but dynamic lifts rely only on layer mass (Positive pixels), suction cups do not enter in equation.

**Reply by bergie5737** [reply]
Apologies for not being clear, but your answer is clear that the suction cup does not contribute to layer mass. 
Would you agree in cases where a large suction cup is that can't be removed, best is to manually slow the lift speed down? 

I also did consider lifting clear out of the vat to let the resin run out and form and "air trap" for the next layer, but not sure that would be helpful.

**Reply by sn4k3** [reply]
For dynamic lifts you should know your limits, when fep releases at most area, and then use that + a margin as the safe max lift height. Also tight FEP should not be problematic. On my printer it detach at +/- 0.6mm, i run my prints with 2mm without care for suction cups. But that will depend on printer robusteness.
It's all about testing, i can't tell you what will be best, my suggestion is try and learn from errors, in other words, do not attempt to fix a problem that dont exists yet, when they come you analayse and learn from them and adjust whatever is necessary.

Anyway, using a slow feedrate will not hurt either if you want to play in the safe side
